andela/northstar-backend
# #168191152 fix test script

### What does this PR do?
- It fixes the test script and ensures tests run on Windows, Linux and Mac Operating Systems

### How can this be manually tested?
-  Clone or download this repository
-  On your terminal, `cd` into the application's directory and run: `git branch -r` to get all the branches from the origin
- Run `git checkout -b bg-test-168191152 origin/bg-test-168191152` to have a copy of this branch locally
-  Set the following environment variables in your `.env` file:
```
DB_PASSWORD_DEVELOPMENT // db password
DB_NAME_DEVELOPMENT // db name
DB_HOST_DEVELOPMENT // host: If  not set, this defaults to 'localhost'
DB_DIALECT=postgres // If not set, this defaults to 'postgres'
```

- If you have not, install [Postgresql](https://www.postgresql.org/download)
- Run `npm run test`
- Your test script should run


#### Any background context you want to provide?
- A couple of tests will fail and ten will pass. You don't have to worry about this. It has nothing to do with this Pull Request.

### What are the relevant Pivotal Tacker stories?
[#168191152 ](https://www.pivotaltracker.com/story/show/168191152)
